nemo/collections/nlp/modules/common/common_utils.py

```diff
@@ -37,7 +37,10 @@ def get_pretrained_lm_models_list() -> List[str]:
 
 
 def get_pretrained_lm_model(
-    pretrained_model_name: str, config_file: Optional[str] = None, checkpoint_file: Optional[str] = None
+    pretrained_model_name: str,
+    config_dict: Optional[dict] = None,
```

Comment:
could you put add @ekmb to this PR, since you are proposing a change in nlp common collection. otherwise i would suggest to keep this PR only about bert and necessary changes, and collection improvement proposals in a separate one where you also add other people as reviewer

Reply:
@ericharper why do we need config_dict=config_dict in there? what's the benefit? having both config_dict and config_file adds complexity. I agree with Yang, let's keep the PR focused on adding hydra to the scripts

Reply:
I already requested her review.

Reply:
@ekmb I reducing complexity with this change. Rather than keeping a seperate .json file, we can put the HF BERT config in the .yaml.